homeassistant/components/mqtt/lock.py

```diff
@@ -286,5 +299,5 @@ async def async_open(self, **kwargs: Any) -> None:
         )
         if self._optimistic:
             # Optimistically assume that the lock unlocks when opened.
-            self._attr_is_locked = False
+            self._attr_is_open = True
```

Comment:
Is this a breaking change?

Reply:
Strictly you are right, added a breaking change section. Under ideal conditions a locks state is updated by the device (`pessimistic` mode).